components/playlist/playlist_util.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/playlist/playlist_util.h"
+
+#include "base/strings/utf_string_conversions.h"
+
+namespace playlist {
+
+std::string ConvertFilePathToUTF8(const base::FilePath& path) {
+#if defined(OS_WIN)
+  return base::UTF16ToUTF8(path.value());
```

Comment:
1) isn't it `FilePath::AsUTF8Unsafe()`? It's pretty widely used
2) I see only one call, probably we could just move it closer to it and avoid adding new files

Reply:
Changed to use `AsUTF8Unsafe()` and `ConvertFilePathToUTF8` has three callers.